scripts/prep-dev

```diff
@@ -0,0 +1,31 @@
+#!/bin/bash
+# Developer-oriented utility script for deploying Saltstack config
+# files for the SecureDrop Workstation dev env. Installs the latest
+# locally built RPM in order to deploy the Salt config.
+set -e
+set -u
+set -o pipefail
+
+
+# The dest directory in dom0 is not customizable.
+dom0_dev_dir="$HOME/securedrop-workstation"
+
+function find_latest_rpm() {
+    find "${dom0_dev_dir}/rpm-build/RPMS/" -type f -iname '*.rpm' | sort -n | tail -n 1
```

Comment:
Could it make sense to prioritize the most recently built file (by timestamp) instead of going by version number? That way, if a `make clone` run was successful, you'd always know that you'd get exactly the version that was just built. I can see the kind of issue I experienced during review (with an old test RPM taking precedence over a more recent version) coming up again in practice, as we may sometimes want to bump the RPM version temporarily as part of a test plan as we did with #596.

Reply:
> Could it make sense to prioritize the most recently built file (by timestamp) instead of going by version number?

Great suggestion! Added.